apps/server/src/modules/transfer/export/app-formatter/app-formatter.ts

```diff
@@ -82,7 +93,7 @@ export class AppFormatter {
         type: resource.type,
         ref: this.resourceTypeToRef.get(resource.type),
       });
-      exportedResources.push(this.exporter.resource(resource, exportContext));
+      exportedResources.push(exportedResource);
```

Comment:
This will avoid calling the plugin's export resource method twice while exporting the application.